src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const elements = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. The variable name 'elements' could be more descriptive. Consider renaming it to reflect that it contains style declarations, such as 'styleDeclarations' or 'styleEntries'.